File: SpecialEducationPlanning/SpecialEducationPlanning.Web/src/app/shared/models/Education-specification-templates.model.ts
import { HousingSpecification } from "./housing-specification.model";
import { Plan } from "./plan";

export interface HousingSpecificationTemplates{
    housingSpecificationId: number;
    planId: number;
    housingSpecificationModel?: HousingSpecification;
    housingSpecification?: HousingSpecification;
    plan: Plan;
    id: number;
}

File: SpecialEducationPlanning/SpecialEducationPlanning.Business/Model/Project/EducationSpecificationTemplatesModel.cs
using Koa.Domain;

namespace SpecialEducationPlanning
.Business.Model;

public class HousingSpecificationTemplatesModel : BaseModel<int>
{
    public int HousingSpecificationId { get; set; }
    public int PlanId { get; set; }
    public HousingSpecificationModel HousingSpecificationModel { get; set; }
    public PlanModel Plan { get; set; }
}
